### DISCUSSION TITLE
Custom transition when presented fullScreenCover

### DISCUSSION THREAD

**Discussion by bsv-doanhnguyen** [SwiftfulThinking/SwiftfulRouting#68] [Q&A]
How to custom transition when set .fullScreenCover? I try set transition left to right in FullScreenCoverViewModifier but not working. Do you have any solutions to handle this issue?

**Comment by SwiftfulThinking** [comment]
Transitions are different from FullScreenCover. We cannot customize the FullScreenCover, although you can use .navigationTransition in iOS 18.
